includes/rest-api/class-wp-rest-stories-controller.php

```diff
@@ -0,0 +1,131 @@
+			return $prepared_story;
+		}
+
+		if ( isset( $request['content_filtered'] ) ) {
+			$prepared_story->post_content_filtered = $request['content_filtered'];
+			if ( isset( $request['content_filtered']['raw'] ) ) {
+				$prepared_story->post_content_filtered = $request['content_filtered']['raw'];
+			}
+			$prepared_story->post_content_filtered = maybe_serialize( $prepared_story->post_content_filtered );
+		}
+
+
+		return $prepared_story;
+	}
+
+	/**
+	 * Prepares a single story output for response. Add post_content_filtered field to output.
+	 *
+	 *
+	 * @param WP_Post $post Post object.
+	 * @param WP_REST_Request $request Request object.
+	 *
+	 * @return WP_REST_Response Response object.
+	 */
+	public function prepare_item_for_response( $post, $request ) {
+		$response = parent::prepare_item_for_response( $post, $request );
+		$fields   = $this->get_fields_for_response( $request );
+		$data     = $response->get_data();
+		$schema   = $this->get_item_schema();
+
+		if ( in_array( 'content_filtered', $fields, true ) ) {
+			$post_content_filtered    = maybe_unserialize( $post->post_content_filtered );
+			$data['content_filtered'] = array(
+				'raw' => rest_sanitize_value_from_schema( $post_content_filtered, $schema['properties']['content_filtered']['properties']['raw'] ),
+			);
+		}
+
+		$context = ! empty( $request['context'] ) ? $request['context'] : 'view';
+		$data    = $this->filter_response_by_context( $data, $context );
+		$links   = $response->get_links();
+		// Wrap the data in a response object.
+		$response = rest_ensure_response( $data );
+		foreach ( $links as $rel => $rel_links ) {
+			foreach ( $rel_links as $link ) {
+				$response->add_link( $rel, $link['href'], $link['attributes'] );
+			}
+		}
+
+		/**
+		 * Filters the post data for a response.
+		 *
+		 * The dynamic portion of the hook name, `$this->post_type`, refers to the post type slug.
+		 *
+		 *
+		 * @param WP_REST_Response $response The response object.
+		 * @param WP_Post $post Post object.
+		 * @param WP_REST_Request $request Request object.
+		 */
+		return apply_filters( "rest_prepare_{$this->post_type}", $response, $post, $request );
+	}
+
+	/**
+	 * Retrieves the attachment's schema, conforming to JSON Schema.
+	 *
+	 *
+	 * @return array Item schema as an array.
+	 */
+	public function get_item_schema() {
+		if ( $this->schema ) {
+			return $this->add_additional_fields_schema( $this->schema );
+		}
+		$schema = parent::get_item_schema();
+
+		$schema['properties']['content_filtered'] = array(
+			'description' => __( 'The post content filtered for the object.', 'amp' ),
+			'type'        => 'object',
+			'context'     => array( 'view', 'edit', 'embed' ),
+			'arg_options' => array(
+				'sanitize_callback' => null, // Note: sanitization implemented in self::prepare_item_for_database()
+				'validate_callback' => null, // Note: validation implemented in self::prepare_item_for_database()
+			),
+			'properties'  => array(
+				'raw' => array(
```

Comment:
Do we need `raw`? Since there is only one property, couldn't this level be removed?

Reply:
This keeps the endpoint inline with other core endpoints. Both `title` and `content` both follow this pattern.

Reply:
Only because they get `rendered` output that needs to be sent. There will never be a `rendered` field for this `content_filtered` because it is essentially the `content.rendered`.

I do not think it make sense to include `raw` for this field.

Reply:
It is the pattern the REST API uses, why diverge from that?

Reply:
The REST API only uses `raw`/`rendered` for fields that have PHP filters applying on the underlying value. There is no `raw`/`rendered` for fields that don't have PHP filters applying (more or less), e.g. `status`, `date`, `link`, `featured_media`. 

Because there is no filters applying on this field, then no `raw` should be included.

Reply:
I was trying to give us an option for us to have a rendered version in the future. But we can always change it later if required. 

Changed in https://github.com/ampproject/amp-wp/pull/3871/commits/25c14233ebe86581b6ed385871fe85ef7784f669